deploy/dracon/chart/values.yaml

```diff
@@ -64,7 +64,14 @@ enrichmentDB:
   # enrichment database
   connectionStr: ""
 
+dracon-components:
+  # if you're developing components
+  #  set this to false to install your own components
+  enabled: true
+
 global:
   image:
     # registry to use for all 
-    registry: ""
+    registry: "ghcr.io"
```

Comment:
all our images already have a default value for the registry in the Helm template, I don't think we need this here or if we put it here, we should remove the use of the `default` function in the Helm templates